apps/wpcom-block-editor/src/wpcom/features/redirect-onboarding-user-after-publishing-post.js

```diff
@@ -29,9 +37,8 @@ export function redirectOnboardingUserAfterPublishingPost() {
 	const unsubscribe = subscribe( () => {
 		const isSavingPost = select( 'core/editor' ).isSavingPost();
 		const isCurrentPostPublished = select( 'core/editor' ).isCurrentPostPublished();
-		const getCurrentPostRevisionsCount = select( 'core/editor' ).getCurrentPostRevisionsCount();
```

Comment:
Since we now allow user save and refresh their post, we don't need to restrict its revision anymore.